npmrc

```diff
@@ -0,0 +1,3 @@
+# Setup NPM auth by adding auth into into .npmrc
+email=teamcity.ciena.com
+registry=https://artifactory.ciena.com/api/npm/blueplanet-npm
```

Comment:
true. However it is necessary for it to work. Thought I'd leave this in for now and figure out a Better Way later on. Do you have any suggestions?